docs/UserGuide.md

```diff
@@ -79,20 +93,25 @@ Examples:
 * `help add`
 * `help edit`
 
-### Adding a person: `add`
+### Adding a person : `add`
```

Comment:
Think we should standardise this to no whitespace between the colon and "person"